pkg/utils/helm/helm.go

```diff
@@ -70,12 +71,13 @@ func GetChartVersion(chart string) (version string, err error) {
 	// }
 
 	// 2. prepare the arguments
-	args := []string{binary, "inspect", "chart", chart,
+	args := []string{helmCmd[0], "inspect", "chart", chart,
 		"|", "grep", "version:"}
 	log.V(1).Info("Exec bash -c", "args", args)
 
 	// cmd := exec.Command("bash", "-c", strings.Join(args, " "))
-	cmd, err := utils.PipeCommand("bash", "-c", strings.Join(args, " "))
+	// TODO: This should be simplified to a no-bash version
+	cmd, err := security.ShellCommand("bash", "-c", strings.Join(args, " "))
```

Comment:
How about using an unified entry like `cmdguard.Command` for pipeCommand and simpleCommand?

Reply:
Turned out that this is in a unused function. Already removed it.